### PR TITLE
 Book List Manager: A complete website with navigation

### PR DESCRIPTION
**Pull request by Titus-Kiplagat**
- [x] Modified the book list manager application to have:

- A Navigation bar.
- Three content sections:
  - Books list.
  - Add book form.
  - Contact info.
 
- [x] Made sure that when you click on any of the links, the main area changes to display only one of the three content sections.

### CODE REVIEW

File: modules/handleNavLinkClick.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+        section.style.display = 'none';
+      });
+
+      document.querySelector(target).style.display = 'block';
+      window.location.href = target;
+    });
+  });
+};
+
+export default handleNavLinkClick;
```
Comment (Ibizugbe) [inline modules/handleNavLinkClick.js:17]: It is a requirement that when a link on the nav bar is clicked, the right section of the page is displayed😉. but currently, the links on your navigation are not working fine. kindly fix it😉.
image

You can take a look at[ this link ](https://www.loom.com/share/2a947b96e4d94c43b3f413aa065002d7?sid=6dc30b11-aafc-4655-a846-5b19d0a72051)for a piece of video evidence where I tested it.

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @Team👋🏼👋🏼👋🏼, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅✅

- [x] Linters are passing👍🏼.
- [x] Home page matches the design👍🏼.
- [x] Webpage has a navigation bar👍🏼.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/handleNavLinkClick.js (hunk 1 above)
Comment (Titus-Kiplagat) [reply modules/handleNavLinkClick.js:17]: @Ibizugbe 
Firstly, thank you for reviewing my code. I would like to report that I've successfully implemented the navigation links functionality in the navbar. Now, when users click on the navigation links, they are correctly redirected to the corresponding page content. I've thoroughly tested this on my end and can confirm that it's working as intended.

To provide you with a visual demonstration, I've prepared a live demo showcasing the navigation links in action. You can access the demo through the following [link](https://www.loom.com/share/a6c8870ec5284306b537c79a221779cd?sid=52063fde-4fb0-4521-8bd3-a3144e65ef68).

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @Titus-Kiplagat, and @hamzaw9👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**